hydra_genetics/utils/io/reference.py

```
@@ -134,39 +138,59 @@ def fetch_reference_data(validation_data, output_dir,
+            for i in range(10):
+                r = s.get(target_url, stream=True, allow_redirects=True, timeout=60)
+                if r.status_code == 202:
+                    logging.info(f"Figshare is preparing file (202). Waiting 10s... (Attempt {i+1})")
+                    r.close()
+                    time.sleep(10)
+                    continue
+                try:
+                    r.raise_for_status()
+                except HTTPError as e:
+                    e.status = e.response.status_code
+                    raise e
```

Ruff (0.14.13)

Use `raise` without specifying exception name

Remove exception name

(TRY201)